src/__tests__/OrsDirections.cy.js

```diff
@@ -47,6 +93,55 @@ describe('Test Directions', () => {
         expect(json1['metadata']['query']['profile']).to.equal('cycling-regular')
         expect(json2['metadata']['query']['profile']).to.equal('foot-walking')
       })
+
+      it('sets customHeaders in request', (done) => {
+        orsDirections.calculate({
+          coordinates: [[8.690958, 49.404662], [8.687868, 49.390139]],
+          profile: 'driving-car',
+          format: 'geojson',
+          customHeaders: {'Accept': 'application/json'}
+        }).catch((err) => {
+          expect(err.message.length).to.be.greaterThan(0)
+          expect(err.response.accepted).to.equal(false)
+          expect(err.response.header['content-type']).to.equal('application/json')
+          done()
```

Comment:
@EmmaHegarty this now tests, not the header of the request but the response, which will be always `application/json` regardless of what is passed in the passed header.

Take a look at how cypress can [intercept](https://docs.cypress.io/api/commands/intercept) made requests and handle testing the request, response or even return something different :)